src/openxr/OpenXRApi.cpp

```diff
@@ -2420,10 +2420,16 @@ bool OpenXRApi::on_state_idle() {
 	for (XRExtensionWrapper *wrapper : registered_extension_wrappers) {
 		wrapper->on_state_idle();
 	}
+
+	emit_plugin_signal(SIGNAL_SESSION_IDLE);
+
 	return true;
 }
 
 bool OpenXRApi::on_state_ready() {
+
+	emit_plugin_signal(SIGNAL_SESSION_READY);
+
```

Comment:
We are already emitting `SIGNAL_SESSION_BEGUN` on state ready, does it makes sense to have two signals here?